config/initializers/sidekiq.rb

```diff
@@ -3,11 +3,7 @@
 password = ENV['REDIS_PASSWORD']
 redis_opts = {}
 
-if ENV['CI']
```

Comment:
Sorry, did we need this before? I got the tests passing here by changing it @SamRaymer